styles.css

```diff
@@ -1297,6 +1292,208 @@ button {
   top: 6290px;
 }
 
+.github-icon {
+  width: 20px;
+  height: 15px;
+  align-items: flex-end;
+}
+
+.popup-btn:hover {
+  background-color: rebeccapurple;
+  color: #fff;
+}
+
+.project-detail-image {
+  width: 100%;
+  height: 220px;
+}
+
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  height: 100%;
+
+  /* bottom: -200%; */
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 4px;
+  z-index: 101;
+  background-color: white;
+  max-height: 99%;
+  width: 97%;
+}
+
+.modal.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-header {
+  padding: 10px 15px;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.modal-header img {
+  max-width: 100%;
+  max-height: 500px;
+}
+
+.modal-header .title {
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.modal-header .close-button {
+  cursor: pointer;
+  height: 26px;
+  width: 26px;
+  border: none;
+  border-radius: 8px;
+  background: none;
+  font-size: 1.25rem;
+  font-weight: bold;
+  margin-bottom: 10px;
+  color: #67798e;
+  position: absolute;
+  top: 15px;
+  right: 15px;
+}
+
+.modal-body {
+  padding: 10px 15px;
+}
+
+#overlay {
+  backdrop-filter: blur(8px);
+  position: fixed;
+  opacity: 0;
+  transition: 200ms ease-in-out;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  pointer-events: none;
+  z-index: 100;
+}
+
+#overlay.active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.modal-hdr-btn {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.modal-hdr-btn h4 {
+  margin: 0;
+  color: #172b4d;
+}
+
+.modal-hdr-btn div {
+  display: flex;
+  position: absolute;
+  bottom: 3px;
+}
```

Comment:
Great job team 🤝

- [ ] You may kindly fix the buttons at the bottom of your mobile popup. These buttons are overlapping content on the page which is not how they are supposed to behave.
- [ ] You may also fix the image size as it appears to be smaller than the one provided in the Figma template.
![mobile popup](https://user-images.githubusercontent.com/63917725/180295968-afaf8bb3-27a8-4b55-891a-2092874d4646.PNG)

Reply:
- [ ] Resolved by margin